CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# dbt_snapchat_ads v0.9.0
+[PR #34](https://github.com/fivetran/dbt_snapchat_ads/pull/34) introduces the following updates:
+
+## Schema Updates
+
+**7 total changes • 0 breaking changes**
+| **Data Model** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| [snapchat_ads__campaign_country_report](https://fivetran.github.io/dbt_snapchat_ads/#!/model/model.snapchat_ads.snapchat_ads__campaign_country_report) | New Transforms Model |   |   |  Each record represents the daily performance of a Snapchat campaign per country.  |
+| [snapchat_ads__campaign_region_report]((https://fivetran.github.io/dbt_snapchat_ads/#!/model/model.snapchat_ads.snapchat_ads__campaign_region_report)) | New Transforms Model |   |   |  Each record represents the daily performance of a Snapchat campaign per region.  |
+| `stg_snapchat_ads__campaign_geo_country_daily_report` | New Staging Model |   |   |  Uses `campaign_geo_country_daily_report` source table  |
```

Comment:
Request to link to the DAG for all models.

Reply:
updated